packages/@aws-cdk/aws-pipes-targets-alpha/lib/firehose.ts

```diff
@@ -0,0 +1,43 @@
+import { IInputTransformation, IPipe, ITarget, TargetConfig } from '@aws-cdk/aws-pipes-alpha';
+import { IRole } from 'aws-cdk-lib/aws-iam';
+import { IDeliveryStream } from 'aws-cdk-lib/aws-kinesisfirehose';
+
+/**
+ * Amazon Data Firehose target properties.
+ */
+export interface FirehoseTargetParameters {
+  /**
+   * The input transformation to apply to the message before sending it to the target.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargetparameters.html#cfn-pipes-pipe-pipetargetparameters-inputtemplate
+   * @default - none
+   */
+  readonly inputTransformation?: IInputTransformation;
+}
+
+/**
+ * An EventBridge Pipes target that sends messages to an Amazon Data Firehose delivery stream.
+ */
+export class FirehoseTarget implements ITarget {
+  private stream: IDeliveryStream;
+  private streamParameters: FirehoseTargetParameters;
```

Comment:
nit: to differentiate the Firehose Delivery Stream from the Kinesis Data Stream, can we use `deliveryStream` over `stream` wherever possible?
```suggestion
  private deliveryStream: IDeliveryStream;
  private deliveryStreamParameters: FirehoseTargetParameters;
```